pipelines/ci-kubernetes-pipeline.yml

```diff
@@ -20,6 +20,21 @@ resources:
     private_key: ((github-private-key))
     paths: [handlers/*]
 
+- name: acceptance-tests-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-acceptance-tests.git
+    private_key: ((github-private-key))
+    paths: [kubernetes.env, tasks/*]
+    branch: run-acceptance-tests-in-ci
```

Comment:
we'll need to change this to master when https://github.com/ONSdigital/census-rm-acceptance-tests/pull/22 is merged in